pkg/util/results/result.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package restore
+package results
```

Comment:
There are a few comments should be updated b/c they explicitly mention the struct is for restore only.

Reply:
Done.